package.json

```diff
@@ -17,46 +17,46 @@
   "author": "",
```

Comment:
These changes to package.json are not necessary, as we have npmshrnikwrap, but maybe it is clearer to change package.json as well :)